network-api/networkapi/wagtailpages/templatetags/multi_image_tags.py

```diff
@@ -8,3 +8,16 @@ def optimize_images(image):
     return {
         'unoptimized_image': image
     }
+
+
+@register.simple_tag(name='custom_image_height')
+def custom_image_height(image, height=410):
```

Comment:
@Pomax Not 100% certain this image tag _should_ live in the `multi_image_tag.py` file. I'm happy to move this if you think there's a better place for it.

Reply:
it might actually make more sense to make this just rename this file to `custom_image_tags.py` instead and update settings.py, if it's going to have various image related tags.

Reply:
Smart. I'll do that and replace its `{% load ... %}` values in the templates when I circle back to this PR. Thanks!

Reply:
Changed! Marking as resolved